README.md

```diff
@@ -140,13 +142,15 @@ YV12/I420 as input format for csc/scaling/blending/rotation, etc. on Ubuntu 16.0
 (with kernel 4.10). The issue can be addressed with the kernel patch:
 WaEnableYV12BugFixInHalfSliceChicken7 [commit 0b71cea29fc29bbd8e9dd9c641fee6bd75f6827](https://cgit.freedesktop.org/drm-tip/commit/?id=0b71cea29fc29bbd8e9dd9c641fee6bd75f68274)
 
-3. CNL: HEVC encoding does not support P frame.
+3. KBL/CFL/CNL: HEVC encoding does not support P frame, detail see [issue#206](https://github.com/intel/media-driver/issues/206)/[issue#207](https://github.com/intel/media-driver/issues/207)/[issue#209](https://github.com/intel/media-driver/issues/209), patch in [PR#262](https://github.com/intel/media-driver/pull/262).
 
 4. HuC firmware is needed for AVC low power encoding bitrate control, including CBR, VBR, etc. As of now, HuC firmware support is disabled in Linux kernels by default. Please, refer to i915 kernel mode driver documentation to learn how to enable it. Mind that HuC firmware support presents in the following kernels for the specified platforms:
    * APL, KBL: starting from kernel 4.11
    * CFL: starting from kernel 4.15
 
 5. Restriction in implementation of vaGetImage: Source format (surface) should be same with destination format (image).
 
+6. ICL encoding and decoding require ICL special kernel mode driver ([issue#267](https://github.com/intel/media-driver/issues/267)/[PR#271](https://github.com/intel/media-driver/pull/271)), please refer to i915 kernel mode driver documentation for supporting status.
```

Comment:
"ICL encoding and decoding require special kernel mode driver" - drop 2nd "ICL"...